### PR TITLE
fix(MutualInformationDiagram): Make sure we set the parameters initia…

### PR DESCRIPTION
**Pull request by scottwittenburg**
…lly, not just when fields chang

### CODE REVIEW

File: src/InfoViz/Native/MutualInformationDiagram/index.js [hunk 1]
```diff
@@ -964,6 +964,10 @@ function informationDiagram(publicAPI, model) {
       });
 
     model.subscriptions.push(model.mutualInformationDataSubscription);
+
+    if (model.provider.setMutualInformationParameterNames) {
```
Comment (jourdain) [inline src/InfoViz/Native/MutualInformationDiagram/index.js]: why would you check that since you are in a block that validated it before?

Comment (scottwittenburg) [inline src/InfoViz/Native/MutualInformationDiagram/index.js]: You mean to say that the check whether the provider is a MutualInformationProvider should be good enough?

Comment (jourdain) [inline src/InfoViz/Native/MutualInformationDiagram/index.js]: yes, that was the reason to put that method call within that block.

**Comment by scottwittenburg** [comment]
@jourdain Should be all set now.

**Pull request merged**